components/account/AccountHoverWrapper.vue

```diff
@@ -7,7 +7,9 @@ const props = defineProps<{
   disabled?: boolean
 }>()
 
-const account = props.account || (props.handle ? useAccountByHandle(props.handle!) : undefined)
+const account = computedAsync(async () => (
+  props.account || (props.handle ? await fetchAccountByHandle(props.handle!) : null)
+), null)
```

Comment:
I feel we should avoid doing async fetch here, as it will not use the cache. Instead, we better fix `useAccountByHandle` here.

Reply:
Isn't caching is done inside `fetchAccountByHandle`? https://github.com/elk-zone/elk/blob/d39e23ebff9620ff1ec226dead8d51f3639b33e0/composables/cache.ts#L59-L88

Or you're talking about another cache here?

Also I tried using `useAccountByHandle` earlier, something like
```js
const account = $computed(() => (props.account || props.handle ? useAccountByHandle(props.handle!).value : null))
```
but it doesn't work.